src/sys/reboot.rs

```diff
@@ -0,0 +1,44 @@
+use {Errno, Error, Result};
+use libc;
+use void::Void;
+use std::mem::drop;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
Sure. Is there a general convention about order of stuff in `#derive`? If so, maybe it should be mentioned/documented somewhere?